pkg/controller/command/command_config_bootstrap.go

```diff
@@ -266,3 +268,48 @@ resources:
       public_url: https://{{ .SwiftProxyAddress }}:{{ .SwiftProxyPort }}
     kind: endpoint
 `))
+
+var migrationConfig = template.Must(template.New("").Parse(`
+database:
+  host: {{ .PostgresAddress }}
+  user: {{ .PostgresUser }}
+  password: {{ .PGPassword }}
+  name: {{ .PostgresDBName }}_migrated
+  max_open_conn: 100
+  connection_retries: 10
+  retry_period: 3s
+  replication_status_timeout: 10s
+  debug: false
+
+log_level: debug`))
+
+var migrationScriptConfig = template.Must(template.New("").Parse(`
+#!/usr/bin/env bash
+
+set -e
+export PGPASSWORD={{ .PGPassword }}
+
+# Try to drop old databases the may or may not exists.
+psql -w -h {{ .PostgresAddress }} -U {{ .PostgresUser }} -d {{ .PostgresDBName }} <<END_OF_SQL
+DROP DATABASE {{ .PostgresDBName }}_migrated;
+DROP DATABASE {{ .PostgresDBName }}_backup;
+END_OF_SQL
+
+# Migrate old database dump to new one.
+commandutil migrate --in /backups/db.yml --out /backups/db_migrated.yml
+
+# Create a database for migrated data, initialize it with a new schema.
+createdb -w -h {{ .PostgresAddress }} -U {{ .PostgresUser }} {{ .PostgresDBName }}_migrated
+psql -v ON_ERROR_STOP=ON -w -h {{ .PostgresAddress }} -U {{ .PostgresUser }} -d {{ .PostgresDBName }}_migrated -f /usr/share/contrail/gen_init_psql.sql
+psql -v ON_ERROR_STOP=ON -w -h {{ .PostgresAddress }} -U {{ .PostgresUser }} -d {{ .PostgresDBName }}_migrated -f /usr/share/contrail/init_psql.sql
```

Comment:
init_psql fills only id and ip pools, just a question isn't it handled by `commandutil convert`?

Reply:
As far as I understand it is not handled by `convert` however I think that those pools aren't in use any more.

Reply:
@cijohnson Could you confirm that ^^